json2html/jsonconv.py

```diff
@@ -28,25 +28,34 @@
 
 if sys.version_info[:2] < (3, 0):
     text = unicode
-    text_types = [unicode, str]
+    text_types = (unicode, str)
 else:
     text = str
-    text_types = [str]
+    text_types = (str,)
```

Comment:
otherwise Python won't see this as a tuple. a tuple with only one element always needs to be followed by a comma